pkg/planner/core/find_best_task.go

```diff
@@ -1365,6 +1365,13 @@ func (ds *DataSource) FindBestTask(prop *property.PhysicalProperty, planCounter
 	}()
 
 	cntPlan = 0
+	bestTaskStore := kv.TiKV
```

Comment:
Just a code style issue. Could `kv.UnSpecified` be better?

Reply:
Just want to use kv.TiKV as the default, since there is always some valid plans on TiKV